src/sentry/web/frontend/debug/mail.py

```diff
@@ -210,6 +212,21 @@ def make_performance_event(project):
     return perf_event
 
 
+def make_generic_event(project):
+    occurrence, group_info = process_event_and_issue_occurrence(
+        TEST_ISSUE_OCCURRENCE.to_dict(),
+        {
+            "event_id": uuid.uuid4().hex,
+            "fingerprint": ["group-1"],
+            "project_id": project.id,
+            "timestamp": before_now(minutes=1).isoformat(),
+        },
+    )
+    generic_group = group_info.group
+    generic_group.update(type=GroupType.PROFILE_BLOCKED_THREAD.value)
```

Comment:
This shouldn't be necessary either, the group should already have this type. If it doesn't then we probably have some kind of bug we should look into.